WordPress/Classes/ViewRelated/System/FlingableViewHandler.swift

```diff
@@ -44,12 +44,31 @@ class FlingableViewHandler: NSObject {
 
         super.init()
 
+        let leftSwipe = UISwipeGestureRecognizer(target: self, action: #selector(handleSwipeGesture(_:)))
```

Comment:
I'd quite like it if these swipe gestures could be conditionally enabled – perhaps an `ignoreHorizontalSwipes` property. Otherwise, it limits the ways the fling gesture can be invoked in 'normal' (non-gallery) image view controllers.